bedrock/contentful/api.py

```diff
@@ -550,6 +610,96 @@ def _get_info_data__locale(self, page_type, entry_fields, entry_obj):
             locale = entry_obj.sys["locale"]
         return {"locale": locale}
 
+    def _get_info_data__product_story(self, entry_fields):
+        # this info is shared between /stories landing template and individual story template
+        # note: the colors available in Color App don't exactly match up with our design palette
+        # so we convert the hex to a theme name and set the colors in CSS
+        COLOR_MAP = {
+            "#ffb4db": "light-pink",
+            "#abf2ff": "light-blue",
+            "#3ee0af": "light-green",
+            "#ab71ff": "light-purple",
+            "#ffd567": "light-yellow",
+            "#ff8ac4": "pink",
+            "#01b3f4": "blue",
+            "#005e5e": "green",
+            "#321c64": "purple",
+            "#ffbd4f": "yellow",
+        }
+
+        # required
+        published = entry_fields.get("published")
+
+        # not required
+        contributors = entry_fields.get("contributors") or []
+        related = entry_fields.get("related_stories") or []
+        contributor_entries = [ContentfulPage.client.entry(contributor.id) for contributor in contributors]
+        related_entries = [ContentfulPage.client.entry(story.id) for story in related]
+
+        # todo: add alt text to all images (ensure contentful has specific field)
+        return {
+            "published": published,
+            "published_formatted": datetime.strptime(published.replace("T", ""), "%Y-%m-%d%H:%M%z").strftime("%B %-d, %Y"),
```

Comment:
Suggestion: would `datetime.datetime.fromisoformat(published).strftime("%B %-d, %Y")` do the same here? Is `published` an ISO-format timestamp? (I'd kind of hope it is, given it's from an API response)

Reply:
Much better, thanks 💯